scripts/mixins/families/imp.lua

```diff
@@ -6,9 +6,24 @@ g_mixins.families = g_mixins.families or {}
 g_mixins.families.imp = function(impMob)
 
     -- 20% chance to break horn on critical hit
+    -- Reacquires horn in stages of 30, 35, 40, 45, and a very rare 60 seconds
     impMob:addListener("CRITICAL_TAKE", "IMP_CRITICAL_TAKE", function(mob)
-        if math.random(100) < 20 and mob:getAnimationSub() == 0 then
+        local random = math.random(100)
+        if random < 20 and mob:getAnimationSub() == 0 then
             mob:setAnimationSub(1)
+            if not mob:getLocalVar("hornDisabled") == 1 then
```

Comment:
`not` only works with true/false comparations.

I think `if mob:getLocalVar("hornDisabled") ~= 1` would work.